legal-api/src/legal_api/reports/report.py

```diff
@@ -364,5 +373,6 @@ class ReportMeta:  # pylint: disable=too-few-public-methods
 
     entity_description = {
         'CP': 'cooperative',
-        'BC': 'BC Benefit Company'
+        'BC': 'BC Company',
+        'BEN': 'BC Benefit Company',
```

Comment:
See https://github.com/bcgov/business-filings-ui/blob/master/src/enums/legalNames.ts

Reply:
Hey sev, I don't think that isn't relevant here. This object maps the backend legal type to the description. Unless you're saying we should be exporting the description from the front end, but I would disagree with that because we will be implementing a legal type/class table that we will pull the legal type/description from in the future

Reply:
I guess I can update this to pull the types from our temp class in business

Reply:
Or are you saying that 'cooperative' is an existing bug and it should be 'BC Cooperative'?

Reply:
I was thinking that we would want legalTypes and legalNames to be the same on both sides.

Reply:
Oh okay -- I updated it to pull the value from the LegalTypes object in the backend (it needs to follow that)

Reply:
It should probably be called legalTypes everywhere, as that was the original name in the schema to reduce confusion.
I'd assume legalName was the businesses legal name.

Reply:
In the UI, `legalNames` is an enum to map the legal type name string. FWIW